.github/workflows/core-cicd-tests.yml

```diff
@@ -52,37 +49,23 @@ jobs:
         run: |
           sh -c "$(curl -fsSL https://raw.githubusercontent.com/dotCMS/dot-cicd/${DOT_CICD_BRANCH}/seed/install-dot-cicd.sh)"
         if: env.jobRun == 'true'
-      - name: Check Provider
-        run: |
-          dotcicd/library/checkProvider.sh github
-        if: env.jobRun == 'true'
-      - name: Fail provider
-        run: |
-          echo "jobRun=false" >> $GITHUB_ENV
-        if: env.jobRun == 'true' && failure()
-      - name: Build Tests Base Image
+      - name: Run Unit Tests
         run: |
-          dotcicd/library/pipeline.sh buildTestsBase
+          ../dotcicd/library/pipeline.sh runUnit
         env:
           LICENSE_KEY: ${{ secrets.DOTCMS_LICENSE }}
         if: env.jobRun == 'true'
-      - name: Run Unit Tests
-        run: |
-          dotcicd/library/pipeline.sh runUnit
-        if: env.jobRun == 'true'
   run-curl-tests-job:
     name: Curl Tests
     runs-on: ubuntu-18.04
     env:
-      DOT_CICD_BRANCH: master
```

Comment:
Let's leave this variable since we need to modify it in the create-release-branch script